packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "courage-for-signin-widget",
+  "version": "1.0.0",
+  "description": "generates a Courage version for signin-widget",
+  "main": "src/CourageForSigninWidget.js",
+  "scripts": {
+    "build": "webpack"
+  },
+  "author": "",
+  "license": "MIT",
```

Comment:
we use `Apache 2` in our sign in widget (https://github.com/okta/okta-signin-widget/blob/master/package.json#L6)
Should this be apache too? I'm not 100% sure, maybe @jmelberg-okta knows...

Reply:
IMO lets just keep it simple - We can remove the `author` and `license` fields